components/ws-manager-bridge/src/cluster-service-server.ts

```diff
@@ -150,6 +154,10 @@ export class ClusterService implements IClusterServiceServer {
     }
 
     public update(call: grpc.ServerUnaryCall<UpdateRequest, UpdateResponse>, callback: grpc.sendUnaryData<UpdateResponse>) {
+        const clientName = call.metadata.get("client-name")
+        if (clientName.length != 0) {
+            log.info("audit log: " + clientName[0] + " requested clusters.update")
```

Comment:
I tam not sure if we should add what updates were made here and in other methods.